arm-recoveryservices/2016-06-01/swagger/backup.json

```diff
@@ -0,0 +1,375 @@
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "name": "backupVaultConfigResource",
+            "in": "body",
+            "description": "Backup vault config resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BackupVaultConfigResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/BackupVaultConfigResource"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/Subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupstorageconfig/vaultstorageconfig": {
+      "get": {
+        "tags": [
+          "BackupStorageConfigs"
+        ],
+        "description": "Fetches resource storage config.",
+        "operationId": "BackupStorageConfigs_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/BackupStorageConfigResource"
+            }
+          }
+        },
+        "deprecated": false
+      },
+      "patch": {
+        "tags": [
+          "BackupStorageConfigs"
+        ],
+        "description": "Updates vault storage model type.",
+        "operationId": "BackupStorageConfigs_Update",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupName"
+          },
+          {
+            "$ref": "#/parameters/VaultName"
+          },
+          {
+            "name": "backupStorageConfigResource",
+            "in": "body",
+            "description": "Backup storage config resource.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/BackupStorageConfigResource"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "NoContent"
+          }
+        },
+        "deprecated": false
+      }
+    }
+  },
+  "definitions": {
+    "BackupStorageConfig": {

```

Comment:
Is this fine for you:
BackupStorageConfigResource -> BackupStorageConfig
BackupStorageConfig -> BackupStorageConfigProperties